### PR TITLE
Adds Bullshit Mode var to RAGIN MAGES

### PR DESCRIPTION
**Pull request by Iamgoofball**
This comes with a preset gamemode set to bullshit mode just for badmins who are 2lazy to fuck with the gamemode controller.

Infinity wizards are made.

They spawn in every 50 seconds instead of 150.

The round does not end until badmins end it.

Good luck!

### CODE REVIEW

**Comment by hornygranny** [comment]
:+1:

**Comment by GunHog** [comment]
You parade against memes, and yet here you are adding these kinds of var names. Tsk, Tsk, Goofball. Hilarious update though, ship it.

**Comment by Iamgoofball** [comment]
I parade against non original and non video game memes

File: code/game/gamemodes/wizard/raginmages.dm [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	var/making_mage = 0
 	var/mages_made = 1
 	var/time_checked = 0
+	var/bullshit_mode = 0 // requested by hornygranny
 
 /datum/game_mode/wizard/announce()
 	world << "<B>The current game mode is - Ragin' Mages!</B>"
```
Comment (Dorsisdwarf) [inline code/game/gamemodes/wizard/raginmages.dm]: Is it intentional that the crew doesn't get to know what hellacious fuck-you is imminent?

Comment (optimumtact) [inline code/game/gamemodes/wizard/raginmages.dm]: if admins are toggling this on then they should also be informing crew

**Comment by WJohn** [comment]
Why not just make it a var to edit the spawn rate of mages?

**Comment by Shadowlight213** [comment]
This is the greatest thing ever.

**Comment by Shadowlight213** [comment]
Wouldn't calling the shuttle still send the round?

**Comment by Shadowlight213** [comment]
1 hour left

**Comment by MrPerson** [comment]
Why not just have the time check be a different var? Then you don't need a special var at all

File: code/game/gamemodes/wizard/raginmages.dm (hunk 1 above)
Comment (freerealestate) [inline code/game/gamemodes/wizard/raginmages.dm]: that's optimistic

**Comment by Shadowlight213** [comment]
@Iamgoofball ADD THE VAR

**Comment by Iamgoofball** [comment]
there's your fuckin extra var


bullshit mode stays so that I can eliminate un-needed loops for maximum overhead reduction for bullshit mages

**Comment by Iamgoofball** [comment]
@Razharas @hornygranny @bs12-bot @Cheridan @RemieRichards @Shadowlight213 @MrPerson @MrStonedOne @paprka @paprika @god

**Comment by AnonymousNow** [comment]
This sounds hilarious, and I cannot wait to play eleventy billion of these rounds.

**Pull request merged**